build.sh

```diff
@@ -50,15 +50,20 @@ png2chr() {
 
 # build CHR if it doesnt already exist
 
-if [ "$(find src/chr/*.chr 2>/dev/null | wc -l)" = 0 ]; then
+if [ "$(find src/chr/*.chr 2>/dev/null | wc -l | xargs)" = 0 ]; then
```

Comment:
This was failing on a mac because `wc` on a mac adds leading whitespace.  Found that piping to xargs is a quick way to trim whitespace and not affect anything on linux.